src/window/building/utility.c

```diff
@@ -165,7 +167,9 @@ void window_building_draw_mission_post(building_info_context *c)
     window_building_play_sound(c, "wavs/mission.wav");
     outer_panel_draw(c->x_offset, c->y_offset, c->width_blocks, c->height_blocks);
     lang_text_draw_centered(134, 0, c->x_offset, c->y_offset + 10, 16 * c->width_blocks, FONT_LARGE_BLACK);
-    window_building_draw_description_at(c, 106, 134, 1);
+    window_building_draw_description_at(c, 66, 134, 1);
```

Comment:
Use this code to use the default offset for the description:
```suggestion
    window_building_draw_description(c, 134, 1);
```

Reply:
Done.